src/reactjs/useSyncExternalStore.ts

```diff
@@ -0,0 +1,2 @@
+// @ts-ignore
+export { useSyncExternalStore } from 'use-sync-external-store/shim/index.js'
```

Comment:
ts-ignore should be unneeded ?

Reply:
Yeah, here it is unneeded, only needed for .native.ts